css/components/buttons.css

```diff
@@ -3,15 +3,28 @@
   cursor: pointer;
   border: 0;
   padding: 0.5rem 0.8rem;
+  transition: 0.2s all ease;
   border-radius: 0.25rem;
   font-size: 1rem;
 }
 
+.btn-outline {
+  outline: 0;
+  cursor: pointer;
```

Comment:
you have given the same property to the button element as well as the btn class. 
you have also applied it on the anchor tag as well
unless you are using it elsewhere, you can omit it.

Reply:
I dont want user to enter `btn btn-outline btn-outline-primary`. So doing it this way

Reply:
oh, alright, thanks for explaining